index.html

```diff
@@ -359,6 +359,19 @@ <h3>MathML Element Mappings</h3>
 		    <span class="subrole">AXSubrole: <code>TBD</code></span>
 		  </td>
 		</tr>
+		<tr id="el-phantom">
+		  <th><a data-cite="MathML3/chapter3.html#presm.mphantom">`mphantom`</a></th>
```

Comment:
I might be missing the context, but where it says the default styling must be overridden in order to be exposed?

Reply:
Isn't it something that is defined in "another spec"? I mean isn't that the case that alll "visibility: hidden" node are not exposed by default?

Reply:
In general, in these AAMs, if the role is only mapped in some scenarios, we do list the scenario in the description. For example: https://w3c.github.io/html-aam/#el-aside-ancestorbodymain

Would be nice to see this land because I want to make an editorial change that will cause a conflict with this branch!

Reply:
OK, so the general rule to exclude elements (including the one with visibility: hidden) from the Accessibility tree is here: https://www.w3.org/TR/wai-aria-1.2/#tree_exclusion

`mphatom`'s default "visibility: hidden" style is here: https://w3c.github.io/mathml-core/#making-sub-expressions-invisible-mphantom

This means that by default the `mphantom` element is not included in the accessibility tree. This PR just says to use the same roles as the `<mrow>` element in case the element is included (e.g. when "visibility: hidden" is reverted).

I don't think we need to specify precisely here when the element is included as that's explained in other specs. Otherwise, I guess we would need to be explicit that e.g. MathML elements are excluded if `visibility: hidden`, etc which would be redundant.

However, we can probably add a note that by default `mpantom` has `visibility: hidden` and so is not included in the tree. How do you suggest to do that? HTML AAM has a "comments" row which would work, but the layout is a bit different in MathML AAM. I'm also not sure I understand what you mean with the `<aside>` example?